modules/ROOT/pages/managing-users.adoc

```diff
@@ -90,9 +90,20 @@ If enabled, the SAML assertions sent from the IdP must be encrypted and follow t
 +
 . Sign out of Anypoint Platform, navigate to the sign-on URL you entered in the *Identity Management SAML 2.0* form, then sign in through your identity provider to test the configuration.
 
+== Import SAML Metadata
+
+You can import your identity provider's SAML 2.0 metadata to Anypoint Platform. You can use *Import IdP Metadata* to import an XML file that contains a SAML 2.0 configuration from your identity provider. The XML file's values populate the *Identity Management SAML 2.0* form.
+
+When you upload an XML file to import your metadata:
+
+* The file must be smaller than 16 KB.
+* The file must contain values that correspond to the fields in the *Identity Provider* page.
```

Comment:
I think we can probably remove this bullet.  It's probably more confusing than helpful, since in 98% of cases this file will be generated automatically by an IDP